Rohit/documentation (#54)
JoshLuong/Trippo: USER-DOCS.md

# Trippo User Documentation

> ## Table of Contents
> - [About](#about)
> - [Getting Started](#start)
> - [Adding Destinations](#add-destination)
> - [Making Changes](#make-changes)
> - [Sharing Itineraries](#share)
> - [Contact Us](#contact)

### <a name="about"> About </a>

> Trippo is a collaborative, map-based, suggestive itinerary planner that allows users to create create customizable itineraries while managing their expenses at each destination.

Reviewer: duplicate word, "allows users to create create"

With Trippo, you can plan for your favorite destinations, add collaborators, create budgets, get Yelp® suggested activities, and more!

> Say Goodbye to using traditional documents and spreadsheets, and say Hello to Trippo!

Reviewer: Should goodbye and hello be capitalized here?

Author: Yeah, Josh wrote that in and I think it looks nicer even though it's not grammatical lol. Like a second slogan type of thing

### <a name="add-destination"> Adding Destinations </a>
To begin planning your trip, you must click on the name of your itinerary. If you have multiple itineraries, it may be easier to use the search bar to find the one you want by searching for the itinerary name, or any tags that correspond with your itinerary.

After clicking on the itinerary you wish to add destinations to, you will be redirected to a map page that has a side calendar view of your trip. 

![Map Page](trippo/public/user-docs/3.PNG)

To add a new destination to your trip, use the search bar on the map page to search for your destination. 

Once you click on the pin that appears on your destination, a new form will appear. 
![New Destination Form](trippo/public/user-docs/5.PNG)

> If you had selected a date on your side calendar, the date on the new form will correspond to that date, otherwise it will be set to the first day of your trip. You can also change this date whichever date during your trip that you want to visit this destination.

Reviewer 2: ```suggestion
If you had selected a date on your side calendar, the date on the new form will correspond to that date, otherwise it will be set to the first day of your trip. You can also change the time of this new activity by editing your activities.
```

not sure what was meant!

Author: I meant change the date to any date during your trip..

Author: fixed and commited

> You can select the type of destination to keep your itinerary organized and colour coordinated, set a cost that you expect to spend at this destination, select a time to visit, and add any notes to yourself or collaborators in the Notes field. Once you have filled out your destination information, you can click the "Add" button on the bottom right of the form to save this destination to your itinerary. 

Reviewer 2: ```suggestion
You can select the type of destination to keep your itinerary organized and colour-coordinated, select a time to visit, add any notes in the Notes field, and although not required, set a cost that you expect to spend at this destination (which will later be used to calculate your daily costs in accordance to your budget, if set). Once you have filled out your destination information, you can click the "Add" button on the bottom right of the form to save this destination to your itinerary. 
```

![Side Calendar Day View](trippo/public/user-docs/4.PNG)

> Opening the side calendar and selecting any date on in your trip will highlight the pins on the map that correspond to that days destinations, and show you a day view of any destinations you have added.

Reviewer 2: ```suggestion
Opening the side calendar and selecting any date on your trip will open day view of any activities you have added. It will also highlight the pins on the map that correspond to the currently selected day.
```

> Opening the side calendar and selecting any date on your trip will open day view of any activities you have added. It will also highlight the pins on the map that correspond to the currently selected day.

Reviewer 2: ```suggestion
Opening the side calendar and selecting any date on your trip will open day view of any activities you have added. It will also highlight the pins on the map that correspond to the currently selected day. Click on the name of any activity to get a visual cue of where the marker is on the map, as the marker will show a small popup of the name and date of the activity.
```

![Itinerary Master Plan](trippo/public/user-docs/9.png)

> You can also click on the "Itinerary master plan" button to see a full itinerary of your trip.

Reviewer 2: ```suggestion
You can also click on the "Itinerary master plan" button to see the full itinerary of your trip in a compact, read-only view.
```

### <a name="make-changes"> Making Changes </a>

> To make changes to your destinations, you can open the side calendar on the map page and select a date for which you want to make a change. 

Reviewer 2: ```suggestion
To make changes to your activities, you can open the side calendar on the map page and select a date for which you want to make a change. 
```

Reviewer 2: ignore

> Here, you are able to click on the Edit button, which allows you to change the time, the cost, or the notes corresponding to any destination on that day, or even delete that destination. 

Reviewer 2: ```suggestion
Here, you are able to click on the Edit button, which allows you to change the time, the cost, or the notes corresponding to any activity on that day, or even delete that activity . 
```

Reviewer 2: redact maybe just stick with 'destination', since it's used elsewhere

![Edit Your Day](trippo/public/user-docs/6.PNG)

> Make sure to click the "Done" button when you have finished making changes to your destinations, otherwise they may will not be saved.

Reviewer: another typo, "they may will not be saved"

Opening the navigation bar in the top left and clicking on your Home page will bring you back to your itineraries. 

![Edit Itinerary](trippo/public/user-docs/10.png)

> Here you can also make changed to your overall trip by clicking on the marker in the top right of your itinerary. In this form you can make changes to your existing itinerary, or even delete it. After you have made any changes, make sure to click the "Update" button to save any changes.

Reviewer: typo, "make changed"

### <a name="share"> Sharing Itineraries </a>

To share your itinerary, navigate to the map page with the side calendar view. At the top of the page, you will find a link button next to the name of your itinerary. 

![Shareable Link](trippo/public/user-docs/11.png)

> Clicking on that button will generate a shareable link to your itinerary. Your itinerary will not be editable through this link and will be displayed in a read-only format. To allow others to make edits to your itinerary, you must add their google emails as collaborators and they can access your itinerary once they log in to Trippo.

Reviewer 2: ```suggestion
Clicking on that button will generate a shareable link to your itinerary. Your itinerary will not be editable through this link, and will be displayed in a read-only format. To allow others to make edits to your itinerary, you must add their google emails as collaborators.
```

![Export to PDF](trippo/public/user-docs/8.PNG)

> You can also click the "Export to PDF" button on the top left of your side calendar on the map view to download a PDF version of your itinerary to share with others offline.

Reviewer 2: ```suggestion
You can also click the "Export to PDF" button on the top left of your side calendar on the map view to download a PDF version of your itinerary to share with others offline. Clicking on the "Export to PDF" button will open a pop up that will allow you to input any custom background for the PDF, but you may also choose to use the default background.
```

### <a name="contact"> Contact Us </a>

> For any other questions or concerns regarding Trippo that have not been addressed in these user docs, feel free to contact us at [tripposupport@protonmail.com](mailto:tripposupport@protonmail.com)

Reviewer: might be more professional to replace "these user docs" with "this documentation"?

Author: Thanks, good looking out. Made the above changes
